### PR TITLE
Frozen: Auto Affe-Sog Flow

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15732

*Description of changes:*
* Apply Frozen state manually to store between Affe-Sog flows
* Spacing adjustment for Home Owners in Mhr Information Flow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -260,7 +260,7 @@
 
                 <HomeOwners
                   isMhrTransfer
-                  class="mt-n2"
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue:263]: I'm not sure the original negative margins here but it seemed something was corrected. 
Now applying appropriate spacing.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1306-vcmix1as.web.app

**Review by dimak1 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1306-vcmix1as.web.app

**Pull request merged**